flake8.ini

```diff
@@ -14,8 +14,12 @@ per-file-ignores =
     # imported but unused
     __init__.py: F401
 
-# require code(s) to be included in # noqa comments
+# flake8-noqa: require code(s) to be included in # noqa comments
```

Comment:
This should be it's own commit, or the change should be at least mentioned in the commit message

Reply:
Fixed